charts/cosmotech-business-webapp/README.md

```diff
@@ -20,6 +20,7 @@ Cosmo Tech Businesss Web Application
 | webapp.functions.image.repository | string | `""` | container image to use for webapp functions (e.g. ghcr.io/cosmo-tech/azure-sample-webapp/webapp-functions) |
 | webapp.functions.image.tag | string | `"latest"` | container image tag |
 | webapp.publicUrl | string | `""` | URL path to use as root of the webapp (e.g. /cosmotech-webapp/brewery) |
+| webapp.registrySecret | string | `""` | name of the secret containing registry credentials to pull private images (e.g. cosmotech-business-webapp-brewery-registry) |
```

Comment:
The standard way to expose this info seems to be a list entry name `pullSecrets` at the same level as the associated `image`, `pullPolicy`,... To be honest, on my side I've not made it a list (so just named it `pullSecret`). But at least, its place and name is probably best as `webapp.server.image.pullSecret` and have a separate one for `webapp.functions.image.pullSecret` (to support different secrets for the 2 images)

Reply:
Fixed by fixup commits [47843c45](https://github.com/Cosmo-Tech/helm-charts/pull/7/commits/47843c4553293bf4e5efa9cba3e24b27f371f4e8) and [82b26218](https://github.com/Cosmo-Tech/helm-charts/pull/7/commits/82b2621819dd75fccb7bbad81f7549dad5413ba5)